### PR TITLE
Add next/image docs for omitted props

### PR DESCRIPTION
**Pull request by styfle**
This will make it clear which properties are not allowed

### CODE REVIEW

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| buildDuration | 13.2s | 12.9s | -255ms |
| nodeModulesSize | 84.9 MB | 84.9 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.433 | 2.383 | -0.05 |
| / avg req/sec | 1027.55 | 1048.99 | +21.44 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.296 | 1.291 | -0.01 |
| /error-in-render avg req/sec | 1928.5 | 1935.74 | +7.24 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| 677f882d2ed8..d9fb.js gzip | 12.7 kB | 12.7 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-a715f78..8dec.js gzip | 6.53 kB | 6.53 kB | ✓ |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 58.9 kB | 58.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 8.23 kB | 8.23 kB | ✓ |
| framework.HA..dule.js gzip | 39 kB | 39 kB | ✓ |
| main-1ab332d..dule.js gzip | 5.64 kB | 5.64 kB | ✓ |
| webpack-07c5..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 53.6 kB | 53.6 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| _app-3b0cf13..85f8.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-6f635..c393.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-d4ffc3..9e0f.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-d9a15a0..130a.js gzip | 1.57 kB | 1.57 kB | ✓ |
| routerDirect..924c.js gzip | 284 B | 284 B | ✓ |
| withRouter-7..c13d.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.98 kB | 7.98 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| _app-75d3a82..dule.js gzip | 625 B | 625 B | ✓ |
| _error-f2fcc..dule.js gzip | 2.29 kB | 2.29 kB | ✓ |
| hooks-cbf13f..dule.js gzip | 387 B | 387 B | ✓ |
| index-b9a643..dule.js gzip | 226 B | 226 B | ✓ |
| link-973cce3..dule.js gzip | 1.51 kB | 1.51 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.6 kB | 5.6 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| _buildManifest.js gzip | 321 B | 321 B | ✓ |
| _buildManife..dule.js gzip | 330 B | 330 B | ✓ |
| Overall change | 651 B | 651 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| index.html gzip | 969 B | 969 B | ✓ |
| link.html gzip | 975 B | 975 B | ✓ |
| withRouter.html gzip | 961 B | 961 B | ✓ |
| Overall change | 2.9 kB | 2.9 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| buildDuration | 15s | 14.6s | -377ms |
| nodeModulesSize | 84.9 MB | 84.9 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| 677f882d2ed8..d9fb.js gzip | 12.7 kB | 12.7 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-a715f78..8dec.js gzip | 6.53 kB | 6.53 kB | ✓ |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 58.9 kB | 58.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 8.23 kB | 8.23 kB | ✓ |
| framework.HA..dule.js gzip | 39 kB | 39 kB | ✓ |
| main-1ab332d..dule.js gzip | 5.64 kB | 5.64 kB | ✓ |
| webpack-07c5..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 53.6 kB | 53.6 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| _app-3b0cf13..85f8.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-6f635..c393.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-d4ffc3..9e0f.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-d9a15a0..130a.js gzip | 1.57 kB | 1.57 kB | ✓ |
| routerDirect..924c.js gzip | 284 B | 284 B | ✓ |
| withRouter-7..c13d.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.98 kB | 7.98 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| _app-75d3a82..dule.js gzip | 625 B | 625 B | ✓ |
| _error-f2fcc..dule.js gzip | 2.29 kB | 2.29 kB | ✓ |
| hooks-cbf13f..dule.js gzip | 387 B | 387 B | ✓ |
| index-b9a643..dule.js gzip | 226 B | 226 B | ✓ |
| link-973cce3..dule.js gzip | 1.51 kB | 1.51 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.6 kB | 5.6 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| _buildManifest.js gzip | 321 B | 321 B | ✓ |
| _buildManife..dule.js gzip | 330 B | 330 B | ✓ |
| Overall change | 651 B | 651 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| _error.js | 916 kB | 916 kB | ✓ |
| 404.html | 4.61 kB | 4.61 kB | ✓ |
| hooks.html | 3.86 kB | 3.86 kB | ✓ |
| index.js | 916 kB | 916 kB | ✓ |
| link.js | 975 kB | 975 kB | ✓ |
| routerDirect.js | 967 kB | 967 kB | ✓ |
| withRouter.js | 967 kB | 967 kB | ✓ |
| Overall change | 4.75 MB | 4.75 MB | ✓ |

</details>

</details>
Commit: 6cec79448027808d45cb1cec4c6f87a1481aa85a

File: docs/api-reference/next/image.md [hunk 1]
```diff
@@ -139,7 +139,11 @@ When true, the source image will be served as-is instead of changing quality, si
 
 ## Other Props
 
-All other properties on the `Image` component will be passed to the underlying `img` element, except for `style`. Use `className` instead.
+Other properties on the `Image` component will be passed to the underlying `img` element with the exception of the following:
+
+- `style` is omitted from the `img`, use `className` instead.
+- `srcSet` is omitted from the `img`, use [Device Sizes](/docs/basic-features/image-optimization.md#device-sizes) instead.
+- `decoding` is omitted from the `img`, it is always assigned `decoding=async`.
```
Comment (lfades) [inline docs/api-reference/next/image.md:146]:
```suggestion
Other properties on the `Image` component will be passed to the underlying `img` element with the exception of the following:

- `style`. Use `className` instead.
- `srcSet`. Use [Device Sizes](/docs/basic-features/image-optimization.md#device-sizes) instead.
- `decoding`. It is always assigned to `"async"`.
```

**Review by lfades — APPROVED** [review]

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| buildDuration | 13.8s | 13.6s | -233ms |
| nodeModulesSize | 84.9 MB | 84.9 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.49 | 2.585 | ⚠️ +0.09 |
| / avg req/sec | 1003.94 | 966.99 | ⚠️ -36.95 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.412 | 1.424 | ⚠️ +0.01 |
| /error-in-render avg req/sec | 1770.53 | 1755.21 | ⚠️ -15.32 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| 677f882d2ed8..d9fb.js gzip | 12.7 kB | 12.7 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-a715f78..8dec.js gzip | 6.53 kB | 6.53 kB | ✓ |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 58.9 kB | 58.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 8.23 kB | 8.23 kB | ✓ |
| framework.HA..dule.js gzip | 39 kB | 39 kB | ✓ |
| main-1ab332d..dule.js gzip | 5.64 kB | 5.64 kB | ✓ |
| webpack-07c5..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 53.6 kB | 53.6 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| _app-3b0cf13..85f8.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-6f635..c393.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-d4ffc3..9e0f.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-d9a15a0..130a.js gzip | 1.57 kB | 1.57 kB | ✓ |
| routerDirect..924c.js gzip | 284 B | 284 B | ✓ |
| withRouter-7..c13d.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.98 kB | 7.98 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| _app-75d3a82..dule.js gzip | 625 B | 625 B | ✓ |
| _error-f2fcc..dule.js gzip | 2.29 kB | 2.29 kB | ✓ |
| hooks-cbf13f..dule.js gzip | 387 B | 387 B | ✓ |
| index-b9a643..dule.js gzip | 226 B | 226 B | ✓ |
| link-973cce3..dule.js gzip | 1.51 kB | 1.51 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.6 kB | 5.6 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| _buildManifest.js gzip | 321 B | 321 B | ✓ |
| _buildManife..dule.js gzip | 330 B | 330 B | ✓ |
| Overall change | 651 B | 651 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| index.html gzip | 969 B | 969 B | ✓ |
| link.html gzip | 975 B | 975 B | ✓ |
| withRouter.html gzip | 961 B | 961 B | ✓ |
| Overall change | 2.9 kB | 2.9 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| buildDuration | 15.6s | 15.4s | -242ms |
| nodeModulesSize | 84.9 MB | 84.9 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| 677f882d2ed8..d9fb.js gzip | 12.7 kB | 12.7 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-a715f78..8dec.js gzip | 6.53 kB | 6.53 kB | ✓ |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 58.9 kB | 58.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 8.23 kB | 8.23 kB | ✓ |
| framework.HA..dule.js gzip | 39 kB | 39 kB | ✓ |
| main-1ab332d..dule.js gzip | 5.64 kB | 5.64 kB | ✓ |
| webpack-07c5..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 53.6 kB | 53.6 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| _app-3b0cf13..85f8.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-6f635..c393.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-d4ffc3..9e0f.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-d9a15a0..130a.js gzip | 1.57 kB | 1.57 kB | ✓ |
| routerDirect..924c.js gzip | 284 B | 284 B | ✓ |
| withRouter-7..c13d.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.98 kB | 7.98 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| _app-75d3a82..dule.js gzip | 625 B | 625 B | ✓ |
| _error-f2fcc..dule.js gzip | 2.29 kB | 2.29 kB | ✓ |
| hooks-cbf13f..dule.js gzip | 387 B | 387 B | ✓ |
| index-b9a643..dule.js gzip | 226 B | 226 B | ✓ |
| link-973cce3..dule.js gzip | 1.51 kB | 1.51 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.6 kB | 5.6 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| _buildManifest.js gzip | 321 B | 321 B | ✓ |
| _buildManife..dule.js gzip | 330 B | 330 B | ✓ |
| Overall change | 651 B | 651 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| _error.js | 916 kB | 916 kB | ✓ |
| 404.html | 4.61 kB | 4.61 kB | ✓ |
| hooks.html | 3.86 kB | 3.86 kB | ✓ |
| index.js | 916 kB | 916 kB | ✓ |
| link.js | 975 kB | 975 kB | ✓ |
| routerDirect.js | 967 kB | 967 kB | ✓ |
| withRouter.js | 967 kB | 967 kB | ✓ |
| Overall change | 4.75 MB | 4.75 MB | ✓ |

</details>

</details>
Commit: e539227dc0e3f3311ea3cf5db8e9d4a56cd42938

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| buildDuration | 13.9s | 14s | ⚠️ +87ms |
| nodeModulesSize | 84.9 MB | 84.9 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.629 | 2.698 | ⚠️ +0.07 |
| / avg req/sec | 950.79 | 926.75 | ⚠️ -24.04 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.591 | 1.67 | ⚠️ +0.08 |
| /error-in-render avg req/sec | 1571.22 | 1497.38 | ⚠️ -73.84 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| 677f882d2ed8..d9fb.js gzip | 12.7 kB | 12.7 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-a715f78..8dec.js gzip | 6.53 kB | 6.53 kB | ✓ |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 58.9 kB | 58.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 8.23 kB | 8.23 kB | ✓ |
| framework.HA..dule.js gzip | 39 kB | 39 kB | ✓ |
| main-1ab332d..dule.js gzip | 5.64 kB | 5.64 kB | ✓ |
| webpack-07c5..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 53.6 kB | 53.6 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| _app-3b0cf13..85f8.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-6f635..c393.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-d4ffc3..9e0f.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-d9a15a0..130a.js gzip | 1.57 kB | 1.57 kB | ✓ |
| routerDirect..924c.js gzip | 284 B | 284 B | ✓ |
| withRouter-7..c13d.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.98 kB | 7.98 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| _app-75d3a82..dule.js gzip | 625 B | 625 B | ✓ |
| _error-f2fcc..dule.js gzip | 2.29 kB | 2.29 kB | ✓ |
| hooks-cbf13f..dule.js gzip | 387 B | 387 B | ✓ |
| index-b9a643..dule.js gzip | 226 B | 226 B | ✓ |
| link-973cce3..dule.js gzip | 1.51 kB | 1.51 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.6 kB | 5.6 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| _buildManifest.js gzip | 321 B | 321 B | ✓ |
| _buildManife..dule.js gzip | 330 B | 330 B | ✓ |
| Overall change | 651 B | 651 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| index.html gzip | 969 B | 969 B | ✓ |
| link.html gzip | 975 B | 975 B | ✓ |
| withRouter.html gzip | 961 B | 961 B | ✓ |
| Overall change | 2.9 kB | 2.9 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| buildDuration | 16s | 15.9s | -140ms |
| nodeModulesSize | 84.9 MB | 84.9 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| 677f882d2ed8..d9fb.js gzip | 12.7 kB | 12.7 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-a715f78..8dec.js gzip | 6.53 kB | 6.53 kB | ✓ |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 58.9 kB | 58.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 8.23 kB | 8.23 kB | ✓ |
| framework.HA..dule.js gzip | 39 kB | 39 kB | ✓ |
| main-1ab332d..dule.js gzip | 5.64 kB | 5.64 kB | ✓ |
| webpack-07c5..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 53.6 kB | 53.6 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| _app-3b0cf13..85f8.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-6f635..c393.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-d4ffc3..9e0f.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-d9a15a0..130a.js gzip | 1.57 kB | 1.57 kB | ✓ |
| routerDirect..924c.js gzip | 284 B | 284 B | ✓ |
| withRouter-7..c13d.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.98 kB | 7.98 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| _app-75d3a82..dule.js gzip | 625 B | 625 B | ✓ |
| _error-f2fcc..dule.js gzip | 2.29 kB | 2.29 kB | ✓ |
| hooks-cbf13f..dule.js gzip | 387 B | 387 B | ✓ |
| index-b9a643..dule.js gzip | 226 B | 226 B | ✓ |
| link-973cce3..dule.js gzip | 1.51 kB | 1.51 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.6 kB | 5.6 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| _buildManifest.js gzip | 321 B | 321 B | ✓ |
| _buildManife..dule.js gzip | 330 B | 330 B | ✓ |
| Overall change | 651 B | 651 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | vercel/next.js image-docs-omitted-props | Change |
| - | - | - | - |
| _error.js | 916 kB | 916 kB | ✓ |
| 404.html | 4.61 kB | 4.61 kB | ✓ |
| hooks.html | 3.86 kB | 3.86 kB | ✓ |
| index.js | 916 kB | 916 kB | ✓ |
| link.js | 975 kB | 975 kB | ✓ |
| routerDirect.js | 967 kB | 967 kB | ✓ |
| withRouter.js | 967 kB | 967 kB | ✓ |
| Overall change | 4.75 MB | 4.75 MB | ✓ |

</details>

</details>
Commit: 190a307de046e809ea7cf28f90b866a1c5b71bc1

**Pull request merged**